tests/testapp/tests/helpers.py

```diff
@@ -11,6 +11,7 @@
 from facility_profile.models import InteractionLog
```

Comment:
I had to update a number of the tests to change the mocking, since we now compute the Instance ID as a function of different things. This file is an example of that (most of the change is from indentation).